packages/contracts-bedrock/STYLE_GUIDE.md

```diff
@@ -20,6 +20,7 @@ with additional rules. These are:
 
 We also have the following custom tags:
 
+- `@custom:predeploy`: Add to a contract whenever its been pre-deployed.
```

Comment:
nit
```suggestion
- `@custom:predeploy`: Add to a contract whenever it's been pre-deployed.
```

Reply:
> The difference between its and it's
Here's the rub: its (without an apostrophe) is a possessive pronoun, like his or her, for nouns that don't have a defined gender. In contrast, it's (with an apostrophe) is the shortened form, or contraction, of it is or it has

Seems right?